### DISCUSSION TITLE
Error tag entrega final

### DISCUSSION THREAD

**Discussion by tomaslcarballo** [dds-jv/foro#216] [Consultas de herramientas] [answered]
Hola!
Entregué el proyecto utilizando el comando el comando del readme: mvn clean verify && git tag entrega-final && git push origin HEAD --tags.
Pero tuve que hacer un nuevo push por lo que elimine el tag entrega final con el siguiente comando: git push --delete origin entrega-final.
Se eliminó el tag e hice el nuevo push del proyecto pero ahora cuando quiero entregarlo aparece el siguiente mensaje de error:
fatal: tag ´entrega final´ already exists.
Habra alguna manera de solucionarlo para así poder entregar el TP?

**Comment by tomaslcarballo** [comment, marked as answer]
Ví que la solución ya estaba en el foro.
Ya lo pude solucionar, mil disculpas!

**Reply by nanderson87** [reply]
Genial que pudiste encontrar la solución en el foro. 
Usaste la respuesta de esta discusión [Error al correr línea para entregar el TP](https://github.com/dds-jv/foro/discussions/202)?

**Reply by tomaslcarballo** [reply]
Sisi, use esa solucion :)
